torch/csrc/lazy/core/view_ops/expand.h

```diff
@@ -7,6 +7,9 @@
 namespace torch {
 namespace lazy {
 
+// Expand can be constructed as a regular view op, or a scalar expand.
```

Comment:
expand isn't really a 'regular view op' in the current PR is it? or maybe i'm mistaken

Reply:
I mean those expand ops that are created through `GetIrValueForExpandedScalar`.